tests/common/fixtures/advanced_reboot.py

```diff
@@ -435,6 +435,11 @@ def __fetchTestLogs(self, rebootOper=None):
             os.makedirs(log_dir)
         log_dir = log_dir + "/"
 
+        # Create a sub-directory to store the logs before the reboot happened
+        log_dir_before_reboot = log_dir + "before_reboot/"
```

Comment:
Should `"before_reboot"` be a subdirectory of `log_dir`, or is that name appended to the end of the name of `log_dir`?

Reply:
Never mind, I see line 436 now. It might be better to use `os.path.join` to both visually make it clear that `"before_reboot"` is a subdirectory, and guarantee that it is treated as a subdirectory.